denizen_scripts/survival/repo-link/command_heal.dsc

```diff
@@ -12,8 +10,11 @@ heal_command:
     - if <context.args.is_empty>:
         - heal <player>
         - feed 20 <player>
```

Comment:
The syntax for this command is `feed (<entity>) (amount:<#>) (saturation:<#.#>)`
```suggestion
        - feed amount:20 <player>
```